packages/core/src/fhir-hydration/resources/observation.ts

```diff
@@ -0,0 +1,91 @@
+import { Observation, Quantity } from "@medplum/fhirtypes";
+import convert, { Unit } from "convert-units";
+
+/**
+ * This map is used to normalize unconventional unit names to standard unit names. i.e cel -> C, millimeter to m, etc.
+ */
+const unitNormalizationMap = new Map<string, string>([["cel", "C"]]);
+
+/**
+ * This map is used to convert standard units to preferred units. i.e C -> F, g -> kg, etc.
+ */
+const unitConversionMap = new Map<string, string>([
```

Comment:
convert().possibilities('mass')
// [ 'mcg', 'mg', 'g', 'kg', 'oz', 'lb', 'mt', 't' ]

the library seems to support functionality like that. I think we should use that so we have all our weight possibilities covered

Reply:
The docs dont mention supoort for temperatures
https://www.npmjs.com/package/convert-units

convert().measures()
// [ 'length', 'mass', 'volume' ]

Would want to confirm this library can support the full range of conversions we need, or whether we need to find a different one or supplement it

Reply:
The library also has the toBest function which might be interesting for us. Its possible that there are cases when a value is more appropriately measured in grams than kilograms?

For example neonatal weights are always done in grams not kg: https://www.urmc.rochester.edu/encyclopedia/content.aspx?contenttypeid=90&contentid=P02673

whereas normal people weights are in kg. 

 Anyway not important now and we will cross that road when we get there just food for thought.

Reply:
Great points! Thx for finding those, @jonahkaye !